src/lib/parsers/xmlMessageParser.ts

```diff
@@ -41,20 +67,23 @@ function getMessageXml(message : MessageObject) : xml.Document {
 	// Ensure it is encoded correctly
 	unescape.chars['&amp;'] = '&amp;';
 	let unencoded = unescape(message.html);
+
 	// XML parsing requires root node. If it doesn't exist, add one.
-	if (!unencoded.toLowerCase().startsWith('<p>')) {
+	if (!unencoded.toLowerCase().trim().startsWith('<p>') && !unencoded.toLowerCase().trim().startsWith('<div>')) {
```

Comment:
I hate this so much...

Reply:
Yeah... this shouldn't need to exist but Cisco is dumb.